ext/abp-filter-parser-modified/abp-filter-parser.js

```diff
@@ -1,106 +1,123 @@
-'use strict'
-
-var elementTypes = ['script', 'image', 'stylesheet', 'object', 'xmlhttprequest', 'object-subrequest', 'subdocument', 'ping', 'websocket', 'webrtc', 'document', 'elemhide', 'generichide', 'genericblock', 'popup', 'other']
+"use strict";
+
+var elementTypes = [
+  "script",
+  "image",
+  "stylesheet",
+  "object",
+  "xmlhttprequest",
+  "object-subrequest",
+  "subdocument",
+  "ping",
+  "websocket",
+  "webrtc",
+  "document",
+  "elemhide",
+  "generichide",
+  "genericblock",
+  "popup",
+  "other",
+];
 
 var elementTypesSet = {
-  'script': 1,
-  'image': 2,
-  'stylesheet': 4,
-  'object': 8,
-  'xmlhttprequest': 16,
-  'object-subrequest': 32,
-  'subdocument': 64,
-  'ping': 128,
-  'websocket': 256,
-  'webrtc': 512,
-  'document': 1024,
-  'elemhide': 2048,
-  'generichide': 4096,
-  'genericblock': 8192,
-  'popup': 16384,
-  'other': 32768
-}
-var allElementTypes = 65535
-
-var separatorCharacters = ':?/=^'
+  script: 1,
+  image: 2,
+  stylesheet: 4,
+  object: 8,
+  xmlhttprequest: 16,
+  "object-subrequest": 32,
+  subdocument: 64,
+  ping: 128,
+  websocket: 256,
+  webrtc: 512,
+  document: 1024,
+  elemhide: 2048,
+  generichide: 4096,
+  genericblock: 8192,
+  popup: 16384,
+  other: 32768,
```

Comment:
I'm going through the changes. I don't know a lot about JavaScript, but is it really OK to remove the quotes on (most of) the keys?